test/utils-tests.jl

```diff
@@ -42,6 +42,17 @@ end
          [44.38225395397647, 48.542947077386664, 67.33061196497327])
 end
 
+# Test baryvel
+@testset "baryvel" begin
+    dvelh_o, dvelb_o = @inferred(baryvel(AstroLib.J2000, 1950))
+    @test dvelh_o ≈ [-29.858889109661163, -4.685067254882613, -2.0307823051175067]
```

Comment:
I didn't have a thorough look to the code, but for this function that performs only algebra operations (the only other AstroLib function called is `premat`, which should exactly match the result by IDL AstroLib) I would expect a result very close to IDL.  Instead:
```
GDL> baryvel, 2451545, 1950d0, a, b
GDL> print, a, format = '(f25.20)'
 -29.85888079464890054737
  -4.68457126684387770155
  -2.03056726813395282960
```